packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx

```diff
@@ -76,3 +90,44 @@ export const getDocs: () => JSX.Element = () => {
     </>
   );
 };
+
+export const getFileSharingDocs: () => JSX.Element = () => {
+  return (
+    <>
+      <Title>File Sharing</Title>
+      <SingleLineBetaBanner />
+      <Description>Microsoft Teams enables file attachment sharing with the following permissions:</Description>
+      <Description>- Anyone</Description>
+      <Description>- Individuals within your organization</Description>
+      <Description>- Participants currently in this chat</Description>
+      <Description>- People who already have access</Description>
+      <Description>- Specific individuals you select</Description>
+      <Description>
+        Currently, Azure Communication Services only supports the "Anyone" and "People you choose" options. The “People
+        in your organization”, “People currently in this chat”, and “People with existing access” permissions can only
+        be supported for users in a Teams tenant. To learn more about the Sharing Permission, please refer to the [Limit
+        sharing in Microsoft
+        365](https://learn.microsoft.com/en-us/microsoft-365/solutions/microsoft-365-limit-sharing?view=o365-worldwide).
```

Comment:
Strip out locale (will auto redirect based on browser language), and for some reason MSDocs PR ask to remove query uri parameter:
`https://learn.microsoft.com/microsoft-365/solutions/microsoft-365-limit-sharing`